app/src/main/java/com/mybus/activity/DisplayFaresActivity.java

```diff
@@ -1,11 +1,46 @@
 package com.mybus.activity;
 
+import android.os.Bundle;
+import android.support.v7.widget.LinearLayoutManager;
+import android.support.v7.widget.RecyclerView;
+
 import com.mybus.R;
+import com.mybus.adapter.FareViewAdapter;
+import com.mybus.asynctask.FaresRequestCallback;
+import com.mybus.model.Fare;
+import com.mybus.service.ServiceFacade;
+
+import java.util.List;
+
+import butterknife.Bind;
+import butterknife.ButterKnife;
 
 /**
  * Created by Julian Gonzalez <jgonzalez@devspark.com>
  */
-public class DisplayFaresActivity extends BaseDisplayActivity {
+public class DisplayFaresActivity extends BaseDisplayActivity implements FaresRequestCallback {
+
+    @Bind(R.id.fares_recycler_view)
+    RecyclerView mFaresRecyclerView;
+    private FareViewAdapter mFareViewAdapter;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        ButterKnife.bind(this);
+        mFaresRecyclerView.setHasFixedSize(true);
+        RecyclerView.LayoutManager mLayoutManager = new LinearLayoutManager(this);
+        mFaresRecyclerView.setLayoutManager(mLayoutManager);
```

Comment:
This lines generates the following error: 
> E/RecyclerView﹕ No adapter attached; skipping layout

A solution is set an empty adapter first:
`mFaresRecyclerView.setAdapter(new FareViewAdapter(new ArrayList<Fare>(), this));`

Source: http://stackoverflow.com/questions/29141729/recyclerview-no-adapter-attached-skipping-layout?answertab=votes#tab-top